src/app/shared/page/footer/footer.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="relative text-center">
+  <div
+    class="w-full absolute top-0 left-0 right-0 w-full absolute inset-x-0 top-0 h-2 bg-gradient-to-r from-teal-500 to-blue-800"
+  >
+    <div class="bg-blue-950 mt-2">
```

Comment:
we don't need to positions
```suggestion
<div class="text-center">
  <div class="h-2 bg-gradient-to-r from-teal-500 to-blue-800"></div>
  <div>
   <div class="bg-blue-950">
```

Reply:
do the same in header